DirectProgramming/DPC++/MapReduce/MonteCarloPi/src/monte_carlo_pi.cpp

```diff
@@ -0,0 +1,167 @@
+#include "monte_carlo_pi.hpp"
```

Comment:
Need to clean up include list a bit

Reply:
Hey Joe, could you provide some clarification here?

After formatting with clang-format, this current include list is the result.